components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -3183,14 +3184,14 @@ private void parseAllowCrossTenantIntrospection(OMElement oauthConfigElem) {
                 ConfigElements.ALLOW_CROSS_TENANT_TOKEN_INTROSPECTION));
         if (allowCrossTenantIntrospectionElem != null) {
             allowCrossTenantIntrospection = Boolean.parseBoolean(allowCrossTenantIntrospectionElem.getText());
-        } else {
-            /* If config is null, the property must be set to the default value which is true to
-        ensure backward compatibility. */
-            allowCrossTenantIntrospection = true;
         }
     }
 
-    public boolean isCrossTenantTokenInspectionAllowed() {
+    /**
+     * This method returns the value of the property AllowCrossTenantTokenIntrospection  for the OAuth configuration
```

Comment:
Extra white space between `AllowCrossTenantTokenIntrospection  for the OAuth `

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1934/commits/661cabce816d2cd6190418e7dba7b1332c5a1aaa